### ISSUE TITLE
How to generate data when solving ODE system

### ISSUE THREAD

**Issue by isablanca** [lululxvi/deepxde#354]
Hi, 

  It may be a stupid problem. When I check the example of ode_system.py, I am confused by 
```
data = dde.data.PDE(geom, ode_system, [ic1, ic2], 35, 2, solution=func, num_test=100)
```
where the function of `solution=func' is explicitly expressed by the subroutine  
```
def func(x):
    """
    y1 = sin(x)
    y2 = cos(x)
    """
    return np.hstack((np.sin(x), np.cos(x)))
```

But I think in a more general case, though we know what the explicit expression of ode_system(x,y) is, we often don't know the expression of func(x).
And in such case, how we construct the `data` variable to train the network.

Best regards

**Comment by ArjunNarayanan** [comment]
I think the keyword `solution` is used to pass the known solution function to the solver. In this case, `deepxde` will use this function to perform test on the model. You are right, we often don't know the true solution. In this case, you will not pass any function to the keyword `solution`.

The workflow to use might be:

- Start with a known problem with a known solution that is as close to your unknown problem
- Use this known problem to tune your network hyperparameters (network size, learning rate, optimizer, etc.)
- After tuning your network and achieving desired accuracy, you can deploy on the unknown problem with more confidence

I hope that helps!
